common/core/desktop/src/kmx/kmx_processor.cpp

```diff
@@ -284,6 +308,10 @@ KMX_BOOL KMX_Processor::ProcessGroup(LPGROUP gp, KMX_BOOL *pOutputKeystroke)
 
   m_miniContext[GLOBAL_ContextStackSize-1] = 0;
 
+  if(m_debug_items) {
+    m_debug_items->push_rule_enter(gp, kkp, m_miniContext, m_indexStack);
+  }
```

Comment:
Corresponds to:
https://github.com/keymanapp/keyman/blob/75e47c1ccc36e1698bafa8178acf8140f88e4e2b/windows/src/engine/keyman32/kmprocess.cpp#L439